client/components/ellipsis-menu/style.scss

```diff
@@ -1,7 +1,7 @@
 .gridicon.ellipsis-menu__toggle-icon {
 	transition: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
+}
 
-	.ellipsis-menu.is-menu-visible & {
-		transform: rotate( 90deg );
-	}
+.ellipsis-menu.is-menu-visible .ellipsis-menu__toggle-icon {
```

Comment:
Leaving out `.gridicon` alters the specificity of the rule – seems unlikely to matter but can we be totally confident that it makes no difference in Calypso without testing existing uses?

Reply:
I'm 99% sure it's redundant but you're right, let's not risk it. I've ammended the commit.